### PR TITLE
Add a integration test trigger to dns, ingress-gce

### PR DESCRIPTION
**Pull request by bowei**
This adds a webhook for integration tests. Right now it's just a stub to validate that the hooks are functional.

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -78,6 +78,38 @@ presubmits:
...
         hostPath:
           path: /mnt/disks/ssd0
+  kubernetes/dns:
+  - name: integration-test
+    agent: kubernetes
+    context: integration-test
+    branches:
+    - master
+    rerun_command: "/test integration-test"
+    trigger: "(?m)^/test( all| pull-community-verify),?(\\s+|$)"
```
Comment (BenTheElder) [inline prow/config.yaml]: s/pull-community-verify/integration-test/

File: prow/config.yaml [hunk 2]
```diff
@@ -78,6 +78,38 @@ presubmits:
...
+    always_run: true
+    spec:
+      containers:
+      - image: gcr.io/google_containers/busybox
```
Comment (krzyzacy) [inline prow/config.yaml]: I'll assume you'll need at least run bootstrap.py with one of our base images (kubekins, or gcloud-in-go), and have `--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)` to checkout the PR of your repo

File: prow/config.yaml [hunk 3]
```diff
@@ -78,6 +78,38 @@ presubmits:
...
+        - "/bin/sh"
+        - "-c"
+        - "echo 'this is a placeholder'"
+  kubernetes/ingress-gce:
+  - name: integration-test
+    agent: kubernetes
+    context: integration-test
+    branches:
+    - master
+    rerun_command: "/test integration-test"
```
Comment (BenTheElder) [inline prow/config.yaml]: s/pull-community-verify/integration-test/
Also this should probably have a different name from the DNS one?
Something like `pull-dns-integration-test` and `pull-ingress-gce-integration-test`

**Comment by BenTheElder** [comment]
these also need entries in `jobs/config.json` and you might want to add them to testgrid

**Comment by bowei** [comment]
I'm going to put the test grid in another CL, want to get this working first.

**Comment by krzyzacy** [comment]
ok I think we can try with this
/lgtm

**Pull request merged**